### PR TITLE
fix some formatting issues

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

File: examples/flutter_gallery/lib/gallery/demos.dart [hunk 1]
```diff
@@ -9,8 +9,10 @@ import 'icons.dart';
 
 class GalleryDemoCategory {
   const GalleryDemoCategory._({ this.name, this.icon });
-  @required final String name;
-  @required final IconData icon;
+  @required
```
Comment (goderbauer) [inline examples/flutter_gallery/lib/gallery/demos.dart]: The annotations should be removed from the class members here and instead be added to the named arguments in the constructor.

**Review by goderbauer — APPROVED** [review]
LGTM with one nit.

File: examples/flutter_gallery/lib/gallery/demos.dart (hunk 1 above)
Comment (a14n) [reply examples/flutter_gallery/lib/gallery/demos.dart]: done

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**